keel-sql/src/main/resources/db/changelog/20191002-add-env-notifications.yml

```diff
@@ -0,0 +1,17 @@
+databaseChangeLog:
+  - changeSet:
+      id: add-env-notifications
+      author: emjburns
+      changes:
+        - addColumn:
+            tableName: environment
+            columns:
+              - column:
+                  name: notifications
+                  type: text
+                  constraints:
+                    nullable: true
```

Comment:
I think we can have a future migration that populates this column with `"[]"` where null and adds a nullable false once we are past the point of having to roll back to prior to the existence of this column.